example_scenes.py

```diff
@@ -1,5 +1,6 @@
 from manimlib import *
```

Comment:
Is this an opensource library?

Reply:
why why why

---
**Sema Reaction:** :question: I have a question | **Sema Tags:** Brittle, Maintainable

Reply:
stage test comment 4.12 files 

__
[![sema-logo](undefined/img/sema-tray-logo.gif)](https://semasoftware.com/gh) &nbsp;**Summary:** :question: I have a question